survey/tests/common-UI-tests-helpers.ts

```diff
@@ -629,6 +632,19 @@ export const fillTripsintroSectionTests = ({
         });
     }
 
+    // Test custom widget activePersonTitle with conditional hasHouseholdSize2OrMoreConditional
+    // Test custom widget buttonSwitchPerson
+    /* @link file://./../src/survey/common/conditionals.tsx */
+    if (householdSize === 1) {
+        testHelpers.inputVisibleTest({ context, path: 'activePersonTitle', isVisible: false });
+        testHelpers.inputVisibleTest({ context, path: 'buttonSwitchPerson', isVisible: false });
+    } else {
+        // NOTE: The following visibility tests for 'activePersonTitle' and 'buttonSwitchPerson' fail for some reason...
```

Comment:
That would be a FIXME more than a note, and a sign that probably the above invisibility test simply succeed because well... the widget has the wrong name?